src/de/mod.rs

```diff
@@ -852,6 +852,12 @@ mod tests {
         assert_eq!(serde_json::from_str::<Empty>(r#"{}"#).unwrap(), Empty {});
     }
 
+    #[test]
+    fn unit() {
+        assert_eq!(from_str(r#"null"#), Ok(()));
+        assert_eq!(serde_json::from_str::<()>(r#"null"#).unwrap(), ());
```

Comment:
Could we test this as part of tuple tests with 0, 1, 2 elements, like here?
https://github.com/CosmWasm/serde-json-wasm/blob/master/src/ser/mod.rs#L608-L612

I think it is strange serde makes a difference for the zero element tuple.

Reply:
Makes sense, but those are serialization tests. We currently don't have the same struct for both serialization and deserialization tests.

Reply:
Yeah, right. But we can test deserialiation of the 3 different tuple types as well. We might need to implement non-unit tuple deserialization, but this is probably good to have anyways.

Reply:
Oh, I thought it was already implemented. Will take a look.

Reply:
Done as suggested.